src/Workspaces/Core/Portable/Shared/Utilities/SymbolEquivalenceComparer.EquivalenceVisitor.cs

```diff
@@ -336,6 +336,11 @@ private bool HandleNamedTypesWorker(INamedTypeSymbol x, INamedTypeSymbol y, Dict
             {
                 Debug.Assert(GetTypeKind(x) == GetTypeKind(y));
 
+                if (x.IsTupleType)
+                {
+                    return y.IsTupleType && HandleNamedTypesWorker(x.TupleUnderlyingType, y.TupleUnderlyingType, equivalentTypesWithDifferingAssemblies);
```

Comment:
This reminds me that I forgot to include @CyrusNajmabadi @dotnet/roslyn-ide for review of the two IDE code changes:
1. SymbolEquivalenceComparer.CombineNamedTypeHashCode (tested by ReorderParameters_Cascade_ToImplementedMethod_WithTuples)
2. SymbolEquivalenceComparer.NamedTypesAreEquivalent (tested by TestFixAll_WithTuples)

I'll let Cyrus comment on Aleksey's question: whether `SymbolEquivalenceComparer` could treat a tuple type and its underlying type as equivalent.